### PR TITLE
GF1127-Submit application

### PR DESCRIPTION
**Pull request by venkata999**
Hi, 
* this is for successful submission of an application It shows a application ref number and information on screen.
* I may need to take email id from Kostya (at the moment I hardcoded the email in the code)
  for display purpose. Testers may need to configure test email account when they test this story.
* Jon , please add you UI code to it. I created an entry in the application css file. Please  have a look.

### CODE REVIEW

**Comment by jon-shipley** [comment]
Styling complete on the submit page.

File: src/main/scala/controllers/ApplicationController.scala [hunk 1]
```diff
@@ -92,16 +96,28 @@ class ApplicationController @Inject()(actionHandler: ActionHandler, applications
...
           overview.sections.find(_.sectionNumber == fs.sectionNumber) match {
             case None => Some(SectionError(fs, "Not started"))
             case Some(s) => checkSection(fs, s)
           }
         }
-        Ok(views.html.showApplicationForm(form, overview, opp, sectionErrors))
-      case None => NotFound
+        if(sectionErrors.isEmpty){
+          val emailto = "experiencederic@university.ac.uk"
+          val dtf = DateTimeFormat.forPattern("HH:mm:ss")
```
Comment (DougC) [inline src/main/scala/controllers/ApplicationController.scala]: I know you were talking about the timestamp a lot with the UX team yesterday, but do we really need to display seconds?

**Comment by DougC** [comment]
I've merged from master and resolved the conflicts in ApplicationController that were due to the changes in my PR from yesterday. Once the checks have completed I think this is good to go.

File: src/main/scala/controllers/ApplicationController.scala (hunk 1 above)
Comment (DougC) [inline src/main/scala/controllers/ApplicationController.scala]: This is something we should probably take up with the UX team. I don't want to hold up the merge of this PR, but it's definitely something to track.

**Pull request merged**